# Below are the scores obtained by a student in tests 
# 34,36,36,38,38,39,39,40,40,41,41,41,41,42,42,45,49,56
# ⦁	Find mean, median, variance, standard deviation.
# ⦁	What can we say about the student marks? 

In [1]:
# here we are importing required libraries
import pandas as pd
import numpy as np

In [3]:
marks = pd.Series([34,36,36,38,38,39,39,40,40,41,41,41,41,42,42,45,49,56])

In [4]:
marks

0     34
1     36
2     36
3     38
4     38
5     39
6     39
7     40
8     40
9     41
10    41
11    41
12    41
13    42
14    42
15    45
16    49
17    56
dtype: int64

In [5]:
marks.describe()

count    18.000000
mean     41.000000
std       5.052664
min      34.000000
25%      38.250000
50%      40.500000
75%      41.750000
max      56.000000
dtype: float64

In [10]:
np.round(marks.var(),2)

25.53

# from the above calculations , we can say that the verage marks are about 41 which is almost eqal to median and the standard deviation is 5.05 , which means the marks are not spread far.